### DISCUSSION TITLE
Is it possible to scroll (on error) to the required field in React Native?

### DISCUSSION THREAD

**Discussion by SoyDiego** [react-hook-form/react-hook-form#11201] [Q&A]
I would like to implement scroll to my inputs when the user doesn't fill the fields.
Someone can guide me or give me an example for React Native?

Thanks!

**Comment by mutasim77** [comment]
check this one: #612

**Reply by SoyDiego** [reply]
Thanks. I have read this post but it's applied more to website and not react-native.
I don't know if @nandorojo or @bluebill1049 can help me a little bit because I read in that post they were talking about react-native but I didn't understand what was the best way or If 2023 there are better options o solutions.

Thanks for your time :)

**Reply by imalgrab** [reply]
Hey @SoyDiego did you figure it out yourself? 🤔

**Comment by phuongthuan** [comment]
2026 and does anyone found the solution yet?
